conf/resources.config

```diff
@@ -10,4 +10,5 @@ params {
     vbi2_bed = "${projectDir}/resources/verifybamid/1000g.phase3.100k.b38.vcf.gz.dat.bed"
     vbi2_mean = "${projectDir}/resources/verifybamid/1000g.phase3.100k.b38.vcf.gz.dat.mu"
     autosomes_non_gap_regions = "${projectDir}/resources/autosomes_non_gap_regions.bed"
+    //autosomes_non_gap_regions = "https://github.com/c-BIG/NPM-sample-qc/raw/v${manifest.version}/resources/autosomes_non_gap_regions.bed"
```

Comment:
to remove this line as it is commented out

Reply:
Done